gcp-test/tests/gcp_dns_record_set/variables.tf

```diff
@@ -0,0 +1,95 @@
+
+variable "resource_name" {
+  type        = string
+  default     = "turbot-test-20200125-create-update"
+  description = "Name of the resource used throughout the test."
+}
+
+variable "record_set_name" {
+  type = string
+  default = "test-record.turbot.com."
+}
```

Comment:
Can we use `var.resource_name`?
It will help to avoid the conflicts with existing resource with same name

Reply:
It is different type of configuration rather than the regular one, managed sone dns is required as suffix of the dns_record_set resource name. So it is hardcoded